Overview.bs

```diff
@@ -1646,12 +1646,17 @@ The algorithm:
 
 <dfn abstract-op>Handle errors</dfn>
 
-Coming soon.
-
-<!-- TODO -->
-
-<!-- TODO: we should consider defining a valid encoded IFT font as one that produces a correct font subset for the corresponding
-           definition in the entry. This would likely be the only requirement made for encoders. -->
+If the extending the font subset process has failed with an error then, some of the data within the font may not be fully loaded and as
+a result rendering content which relies on the missing data may result in incorrect renderings. The client may choose to continue using
+the font, but should only use it for the rendering of codepoints, features, and design space that are fully loaded. An element of the
+font can be considered fully loaded if data for that element is present in the font (eg. for codepoints, listed in the cmap) and there are
+no mapping entries in the font subset which reference that element. Rendering of all other content should fallback to a different font
+following normal client fallback logic. Alternatively, after a failure the client may choose to treat the entire font as failed and not
```

Comment:
That gives more flexibility but reduced testability. What is the advantage of rejecting the entire font? What class of error is avoided by doing so?

Reply:
What I intended here is to give a spectrum of options to a client implementation for how to handle failures and let them choose what makes sense given client specific details. However, I think given that the first option is specified as a "should" and makes sense for the vast majority of clients then, we can probably omit specifically calling out the option to reject the whole font. It's technically still allowed if there's a client implementation where it makes sense.

Reply:
Alright, removed that sentence.